mod_bot/conf.py

```diff
@@ -0,0 +1,81 @@
+import os
+import json
+import pprint
+
+def read_config(path):
+    print("reading configuration")
+    if os.path.exists(path) is False:
```

Comment:
maybe we should fail if we can't read a conf? Or set default values?

Reply:
Where should we store the default values ? 
- In a configuration file ? And what if that configuration file cannot be read for whatever reason ?
- Inside the code ? Not a fan of this solution as we'd need to update it everytime some mandatory configuration is added or removed

I can make it fail if it can't read the file because it does not exist, but I prefer the "empty" configuration (where every "get" afterwards will not return anything) rather than having to handle the exception afterwards